### DISCUSSION TITLE
Interface Connections device B always "-"

### DISCUSSION THREAD

**Discussion by peterbaumert** [netbox-community/netbox#10082] [General]
Hi since 3.3.0 the Device B under the Interface connection always only shows "-".
Same for "PDU" and "Outlet" under Power conncetions.
Is this intended and i need to migrate something or is this a bug?

**Comment by candlerb** [comment]
Can you be more specific about what you mean by "the Device B under the Interface connection"?  Can you give the URL path which is in the browser URL bar?

You might be looking at `/dcim/devices/1/interfaces/` or `/dcim/cables/add/` or `/dcim/cables/` or `dcim/cables/3/` or something else I haven't thought of. (And for all of those, the B end looks OK to me).

**Reply by peterbaumert** [reply]
https://demo.netbox.dev/dcim/interface-connections/

This i mean. Sorry forgot to add it before.

**Reply by candlerb** [reply]
OK, then I see the same as you.

3.2.9:
![image](https://user-images.githubusercontent.com/44789/185759414-5947a0e6-4185-4336-a19b-dd945174f991.png)

3.3.0:
![image](https://user-images.githubusercontent.com/44789/185759425-2ea61929-7b10-4102-9450-3f6a8c98f88f.png)

You should open a bug for this.

**Reply by peterbaumert** [reply]
Will do tomorrow when i am at the computer again.

**Reply by candlerb** [reply]
FYI, "Power Connections" are also broken in the same way. (I'd guess "Console connections" too, but I don't have any in my test dataset migrated from 3.2.9)

![image](https://user-images.githubusercontent.com/44789/185784492-9ac7cb2b-6fe8-4b8b-8ba9-f5e81c642321.png)

If I delete a power connection and recreate it, I still see the same in this table.
